src/server/handlers.rs

```diff
@@ -20,9 +20,15 @@ pub mod v1 {
     pub async fn query_invoices<S: Search>(
         options: QueryOptions,
         index: S,
+        accept_header: Option<String>,
     ) -> Result<impl warp::Reply, Infallible> {
         let term = options.query.clone().unwrap_or_default();
         let version = options.version.clone().unwrap_or_default();
+        debug!(
+            "Querying with term {} and version{}",
+            term.clone(),
```

Comment:
These should be added as structured fields rather than string interpolation:

```rust
debug!(%term, %version, "Querying index");
```

Same goes for most of the new `debug`s in this PR